exporter/googlecloudexporter/exporter.go

```diff
@@ -17,16 +17,23 @@ package googlecloudexporter
 import (
 	"context"
 	"fmt"
+	"os"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/pdata/plog"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/collector/pdata/ptrace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
 )
 
+// hostname is the name of the current host
+var hostname, _ = os.Hostname()
```

Comment:
We're ignoring the error here. Are we ok with an empty string if this fails? This shouldn't fail but it's possible.